user-docs/pupil-capture.md

```diff
@@ -93,11 +148,12 @@ In the `Advanced` sub-menu you can set the `sample duration` -- the number of fr
 
 > {{< video-webm src="/videos/calibration/calibration-manual/clb-man.webm" >}}
 
-This method is done with an operator and a subject. It is suited for midrange distances and can accommodate a wide field of view. You need markers made of concentric circles, like the two shown below.
+This method is done with an operator and a subject. It is suited for midrange distances and can accommodate a wide field of view. You need a calibration
+marker made of concentric circles, like the one shown in the video.
```

Comment:
The operator will use a printed calibration marker like the one shown in the video. 

(link to the calibration marker should be here as well)